tldw_Server_API/app/core/Slides/style_packs/technical_grid.css

```
@@ -0,0 +1,13 @@
+.reveal[data-style-pack="technical_grid"] {
+  background-color: var(--surface, #0f172a);
+  background: linear-gradient(90deg, var(--grid, rgba(125, 211, 252, 0.16)) 1px, transparent 1px),
```

Stylelint (17.5.0)

[error] Unexpected shorthand "background" after "background-color" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)